src/instructlab/configuration.py

```diff
@@ -390,6 +501,30 @@ class _train(BaseModel):
 
     additional_args: dict[str, typing.Any]
 
+    @model_validator(mode="before")
+    def fill_defaults(
+        cls, values: typing.Dict[str, typing.Any]
+    ) -> typing.Dict[str, typing.Any]:
+        defaults = {
+            "model_path": DEFAULTS.MODEL_REPO,
+            "data_path": DEFAULTS.DATASETS_DIR,
+            "ckpt_output_dir": DEFAULTS.CHECKPOINTS_DIR,
+            "data_output_dir": DEFAULTS.INTERNAL_DIR,
+            "max_seq_len": 4096,
```

Comment:
same thing as I said before- I think these baseline defaults should come from an internal config file rather than being hard-coded into the config.

Reply:
Disagree.  The code needs to be usable without having any config files.